web/package.json

```diff
@@ -5,41 +5,42 @@
   "description": "Dashboard for Apache APISIX",
   "scripts": {
     "prepare": "cd .. && husky install web/.husky",
-    "analyze": "cross-env ANALYZE=1 yarn run build",
-    "build": "cp -R ./node_modules/monaco-editor ./public/ && umi build",
-    "dev": "yarn run start:dev",
-    "fetch:blocks": "pro fetch-blocks --branch antd@4 && yarn run prettier",
+    "analyze": "cross-env ANALYZE=1 pnpm run build",
+    "build": "cp -R $(find -wholename ./node_modules/.pnpm/\"monaco-editor@*\"/node_modules/monaco-editor) ./public/ && umi build",
```

Comment:
Since pnpm only works on node_ A soft connection is placed in the mouschedule, and it cannot work normally. Therefore, it must be copied from the original address. I used regular expressions to adapt to the changes of Monaco versions in the future